build-system/app.js

```diff
@@ -1331,4 +1331,21 @@ function decryptDocumentKey(encryptedDocumentKey) {
   return parsedJson.key;
 }
 
+// serve local vendor config JSON files
+app.use('(/dist)?/rtv/*/v0/analytics-vendors/:vendor.json', (req, res) => {
+  const {vendor} = req.params;
+  const max = pc.env.SERVE_MODE === 'default' ? '.max' : '';
```

Comment:
`SERVE_MODE` supports three values: 
`default` which serves .max file
`compiled` which serves compile file
and 
`cdn` which serves file from cdn. 

You may also want to add support to `cdn` serve mode. Thanks.

Reply:
If the `SERVE_MODE=cdn`, do the vendor requests go through the local server still? I thought that the requests would go straight to `https://cdn.ampproject.org/`

Reply:
The baseUrl will still be calculated as `localhost:8000, which will go through the local server.

Reply:
done :)